examples/petstore/lib/testdata/doc/openapi.golden.json

```diff
@@ -1424,6 +1424,15 @@
 						}
 					},
 					{
+						"examples": {
```

Comment:
So I think you updated this by hand? The proper way to update this to use `make golden-update`. You should update a route in the `example/petstore/controllers` to show the change.